src/util/make-knex.js

```diff
@@ -35,7 +35,7 @@ module.exports = function makeKnex(client) {
     },
 
     batchInsert: function(table, batch, chunkSize) {
-      chunkSize = _.isNumber(chunkSize) ? chunkSize : 1;
+      chunkSize = (_.isNumber(chunkSize) && chunkSize > 0) ? chunkSize : 1000;
```

Comment:
@rhys-vdw yep, I was thinking about it, but felt lazy to implement it in this pull request. I'll add another for that.